### PR TITLE
[Docs] Add chinese documentation of lidar_sem_seg3d

### PR DESCRIPTION
**Pull request by Xiangxu-0103**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by Xiangxu-0103** [comment]
#1359

File: docs/zh_cn/supported_tasks/lidar_sem_seg3d.md [hunk 1]
```diff
@@ -1 +1,78 @@
-# 基于Lidar的3D语义分割
+# 基于激光雷达的 3D 语义分割
+
+基于激光雷达的 3D 语义分割是 MMDetection3D 支持的最基础的任务之一。它期望给定的模型以激光雷达采集的任意数量的特征点为输入，并预测每个输入点的语义标签。接下来，我们以 ScanNet 数据集上的 PointNet++ (SSG) 为例，展示如何准备数据，在标准的 3D 语义分割基准上训练并测试模型，以及可视化并验证结果。
+
+## 数据准备
+
+最开始，我们需要从 ScanNet [官方网站](http://kaldir.vc.in.tum.de/scannet_benchmark/documentation)下载原始数据。
```
Comment (ZCMax) [inline docs/zh_cn/supported_tasks/lidar_sem_seg3d.md:7]: 最开始 -> 首先

File: docs/zh_cn/supported_tasks/lidar_sem_seg3d.md [hunk 2]
```diff
@@ -1 +1,78 @@
...
+
+由于不同数据集上的原始数据有不同的组织方式，我们通常需要用 pkl 或 json 文件收集有用的数据信息。
```
Comment (ZCMax) [inline docs/zh_cn/supported_tasks/lidar_sem_seg3d.md:9]: 不同数据集的原始数据

File: docs/zh_cn/supported_tasks/lidar_sem_seg3d.md [hunk 3]
```diff
@@ -1 +1,78 @@
...
+
+## 训练
+
+接着，我们将使用提供的配置文件训练 PointNet++ (SSG) 模型。当你使用不同的 GPU 设置进行训练时，你基本上可以按照这个[教程](https://mmdetection3d.readthedocs.io/zh_CN/latest/1_exist_data_model.html#inference-with-existing-models)的示例脚本。假设我们在一台具有 2 块 GPU 的机器上使用分布式训练：
+
+```
+./tools/dist_train.sh configs/pointnet2/pointnet2_ssg_16x2_cosine_200e_scannet_seg-3d-20class.py 2
+```
+
+注意，配置文件名中的 `16x2` 是指训练时用了 2 块 GPU，每块 GPU 上有 16 个样本。如果你的自定义设置不同于此，那么有时候你需要相应的调整学习率。基础的规则可以参考[此处](https://arxiv.org/abs/1706.02677)。
```
Comment (ZCMax) [inline docs/zh_cn/supported_tasks/lidar_sem_seg3d.md:51]: 基本规则

File: docs/zh_cn/supported_tasks/lidar_sem_seg3d.md [hunk 4]
```diff
@@ -1 +1,78 @@
...
+
+## 定量评估
+
+在训练期间，模型权重将会根据配置文件中的 `evaluation = dict(interval=xxx)` 设置被周期性地评估。我们为不同的数据集支持官方评估方案。对于 ScanNet，将使用 20 个类别的平均交并比 (mIoU) 对模型进行评估。评估结果将会被打印到终端中，如下所示：
```
Comment (ZCMax) [inline docs/zh_cn/supported_tasks/lidar_sem_seg3d.md:55]: 我们支持不同数据集的官方评估方案

File: docs/zh_cn/supported_tasks/lidar_sem_seg3d.md [hunk 5]
```diff
@@ -1 +1,78 @@
...
+    work_dirs/pointnet2_ssg/latest.pth --eval mIoU
+```
+
+## 测试与提交
+
+如果你只想在在线基准上进行推理或测试模型性能，你需要将之前评估脚本中的 `--eval mIoU` 替换成 `--format-only`，并将 ScanNet 数据集[配置文件](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/_base_/datasets/scannet_seg-3d-20class.py#L126)中的 `ann_file=data_root + 'scannet_infos_val.pkl'` 变成 `ann_file=data_root + 'scannet_infos_test.pkl'`。记住将 `txt_prefix` 指定为保存测试结果的目录，例如，添加选项 `--eval-options txt_prefix=work_dirs/pointnet2_ssg/test_submission`。在生成结果后，你可以压缩文件夹并上传至 [ScanNet 评估服务器](http://kaldir.vc.in.tum.de/scannet_benchmark/semantic_label_3d)上。
+
+## 定性评估
+
+MMDetection3D 也为可视化提供了通用的工具，以便于我们可以对训练好的模型预测的分割结果有一个直观的感受。你也可以在评估阶段通过设置 `--eval-options 'show=True' 'out_dir=${SHOW_DIR}'` 来在线可视化分割结果，或者使用 `tools/misc/visualize_results.py` 来离线的可视化。此外，我们还提供了脚本 `tools/misc/browse_dataset.py` 用于可视化数据集而不做推理。更多的细节请参考[可视化文档](https://mmdetection3d.readthedocs.io/zh_CN/latest/useful_tools.html#visualization)。
```
Comment (ZCMax) [inline docs/zh_cn/supported_tasks/lidar_sem_seg3d.md:78]: MMDetection3D 还提供了通用的可视化工具

Comment (ZCMax) [inline docs/zh_cn/supported_tasks/lidar_sem_seg3d.md:78]: 离线地进行可视化

**Review by ZCMax — APPROVED** [review]

**Pull request merged**